### DISCUSSION TITLE
How to bundle subfolder structure with correct paths?

### DISCUSSION THREAD

**Discussion by heydavee** [parcel-bundler/parcel#9394] [Q&A]
Hi i am trying to build my project and deploy it but having an issue so far.

This is my project structure:
```
src
 | - index.html
 | - en
 |     |- index.html
 | - images
 |     |- image.png
```

when building 
``
parcel build src/index.html --public-url ./ --dist-dir build --no-optimize`

I get the desired structure in my build folder, but the links in my subfolder are wrong.

../images/image.png

becomes this:

image.png

Any idea how to fix this? Thanks for your help.

**Comment by mischnic** [comment]
There are a few namer plugins listed on https://parceljs.org/plugin-browser/?type=%22namer%22&page=0&filter=%22%22&includeOfficial=false that do this
